src/get-updated-notifications.js

```diff
@@ -0,0 +1,13 @@
+export const getUpdatedNotifications = taskNotifications => {
```

Comment:
Extracted from https://github.com/dhis2/data-administration-app/blob/1f3aca304ae95a6d7a364cafe922cd740746ff97/src/pages/resource-tables/ResourceTables.js#L197-L214 and https://github.com/dhis2/data-administration-app/blob/1f3aca304ae95a6d7a364cafe922cd740746ff97/src/pages/analytics/Analytics.js#L169-L186